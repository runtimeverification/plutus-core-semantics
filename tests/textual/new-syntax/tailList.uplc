(program 0.0.0 [(force (builtin tailList)) (con list(integer) [ 1, 2, 3 ])])