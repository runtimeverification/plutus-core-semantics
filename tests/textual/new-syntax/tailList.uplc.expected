      ( con list ( integer ) [ 2 , 3 , ] ) 